client/src/components/App.jsx

```diff
@@ -1,24 +1,50 @@
+/* eslint-disable import/extensions */
 import React from 'react';
-// eslint-disable-next-line import/extensions
 import ReviewTile from './ReviewTile/ReviewTile.jsx';
 import Overview from './Overview/Overview.jsx';
 import RelatedItems from './RelatedItems/RelatedItems.jsx';
 import QandA from './QandA/QandA.jsx';
 // eslint-disable-next-line import/extensions
 import products from '../../dummydata.js';
+import queries from './queries.js';
 
 class App extends React.Component {
   constructor() {
     super();
-    this.state = { items: products };
+    this.state = {
+      items: products,
+      productList: [],
+      productInfo: {},
+      productStyles: {},
+      relatedProducts: [],
+      currentProduct: 17762,
+    };
+  }
+
+  componentDidMount() {
+    const { currentProduct } = this.state;
+    this.getAll(1, 20, currentProduct);
+  }
+
+  getAll(pageNumber, countNumber, productId) {
```

Comment:
excellent job using promise.all and catching errors. we can build on this by adding http requests for reviews, questions and cart data